### PR TITLE
Initial Files For CDI CTS as a FAT bucket

### PR DESCRIPTION
**Pull request by hutchig**
Signed-off-by: Gordon Hutchison <gordon.hutchison@gmail.com>

I need to do some more work on the TCK runner finding the correct tests
but it does go a good way.

So this is just parked for now as I have been asked to move on to our next
piece of work.

run with 
cd dev 
./gradlew -Dfat.test.mode=experimental com.ibm.ws.cdi.cts_fat_tck:buildandrun

### CODE REVIEW

**Comment by hutchig** [comment]
I found this page good on running PRs locally:
https://help.github.com/articles/checking-out-pull-requests-locally/

**Comment by hutchig** [comment]
Is there a next step for this idea @shighbar @hibell ? or is it basically dead?

**Comment by shighbar** [comment]
Gordon, we've been focused on some other work so at the moment it's still basically dead but I'm still interested in getting it setup to run as a FAT test if we can. I'll bring this up with Gene and Hank next week some time and see if we can carve out some cycles to move this forward again.

File: dev/com.ibm.ws.cdi.cts_fat_tck/fat/src/com/ibm/ws/cdi/cts/tck/CdiCtsTckLauncher.java [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        HashSet<String> versionedLibraries = null;
+        String backStopImpl = null;
+
+        MvnUtils.runTCKMvnCmdWithProps(server, "com.ibm.ws.cdi.cts_fat_tck", this.getClass() + ":launchCdiCtsTck",
+                                       addedProps, versionedLibraries, backStopImpl);
+    }
+
+    @Mode(TestMode.LITE)
+    @Test
+    public void testThatDoesNothingAndCanAlwaysRunAndPass() {
```
Comment (benjamin-confino) [inline dev/com.ibm.ws.cdi.cts_fat_tck/fat/src/com/ibm/ws/cdi/cts/tck/CdiCtsTckLauncher.java]: We already have a libraray method for this. Have a look at com.ibm.ws.microprofile.reactive.streams.operators_fat_tck/fat/src/com/ibm/ws/microprofile/reactive/streams/operators/tck/FATSuite.java for an example.

**Comment by benjamin-confino** [comment]
I'm approving this in the knowledge it is unfinished. As it is marked `@Mode(TestMode.EXPERIMENTAL)` it won't run; delivering it will just ensure the code is safe and accessible.

**Pull request closed without merging**